### ISSUE TITLE
About tuning the pre-trained model for OOD detection

### ISSUE THREAD

**Issue by hongxin001** [hendrycks/pre-training#3]
Hi,

In the paper, it writes: "Without assuming such knowledge, we use the maximum softmax probabilities to score anomalies and show that models which are **pre-trained then tuned** provide superior anomaly scores". So I want to confirm whether you fine-tune the whole model or just the classifier (the last layer). Thank you.

Best Regards,
Hongxin

**Comment by hendrycks** [comment]
If I recall correctly, the whole network is modified.
